include/boost/spirit/home/x3/nonterminal/detail/transform_attribute.hpp

```diff
@@ -22,7 +23,10 @@ namespace boost { namespace spirit { namespace x3
     {
         typedef Transformed type;
 
-        static Transformed pre(Exposed&) { return Transformed(); }
+        static Transformed pre(Exposed&)
+        {
+            return traits::value_initialize<Transformed>::call();
```

Comment:
Hm, I thought `make_attribute` uses `value_initialize`, maybe we do not need it at all. Revert this for now.

Reply:
Qi's `make_attribute` uses `value_initialize`, that's where my confusion came from. Currently I do not know why Qi uses it (from `value_initialized` documentation it looks like it used wrongly), and Qi does not have `value_initialize` so no need to add complexity until there is a real need for it.